meshnets/model/model.py

```diff
@@ -0,0 +1,49 @@
+"""Define the main MeshGraphNet model"""
+
+import numpy as np
+import torch
+
+from meshnets.datatypes.graph import Graph
+from meshnets.model.encoder import GraphEncoder
+from meshnets.model.decoder import GraphDecoder
+from meshnets.model.processor import GraphProcessor
```

Comment:
Also where are the `GraphEncoder` and pther models implemented?

Reply:
I already saw that they are not implemented. That was my fault. However when something is not implemented we shoudl replace the import with:

```python
class GraphEncoder:
    pass
```

Reply:
The meshnets.model.x is an absolute import from the meshnets directory. This allows for example to easily and clearly import between subfolders in the project. I implemented it this way through the project but it can be changed to local imports instead if you think it is better.

For now I will change the "model" directory to "modules" and create the empty class files.

Reply:
> I already saw that they are not implemented. That was my fault. However when something is not implemented we shoudl replace the import with:
> 
> ```python
> class GraphEncoder:
>     pass
> ```

Where am I supposed to include this in general ? I gives pylint errors if the implemented classes are not coherent with the rest of the code (e.g. callable, correct number of arguments, class docstring). That seems very demanding for empty placeholder classes.

Reply:
Please let me know if the way I did it works for you @AugustoPeres !